spec/conjur-openapi.yml

```diff
@@ -0,0 +1,259 @@
+openapi: "3.0.0"
```

Comment:
is this a good time to consider renaming this file? do we need `openapi` in the filename? is there a standard for the main spec file's name?

Reply:
➕

Reply:
Perhaps something like `index.yml`?

Reply:
see https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#documentStructure

> It is RECOMMENDED that the root OpenAPI document be named: openapi.json or openapi.yaml.